docs/claimantmodel/Maps.md

```diff
@@ -0,0 +1,115 @@
+The Log Claimant Model states that the purpose of Log Transparency is to enable discoverability of Claims.
+To understand whether this statement applies to Maps, we need to understand what it is about Logs that enables discoverability.
+
+A Log can be thought of as a Map where the key is the leaf index, and new entries need to be written into the next available leaf.
+Maps and Logs both have a single root hash which precisely commits to key/value data, and given this root hash and a key, there is only one verifiable value for the point lookup.
+The key difference (pun intended) is that in Maps the writer gets to choose any arbitrary key, where in Logs the key is the next available sequence number.
+
+A Log Checkpoint commits to a range of keys by stating the tree size.
+Tree size is the only input required to generate all keys for a log, and it is this property of key enumerability that makes Claim discoverability simple; given a Log Checkpoint, a Verifier can discover all Claims by requesting all entries from [0..treeSize).
+Thus providing the Verifier is getting recent Checkpoints and is seeing Checkpoints consistent with the Believer, each and every Claim will be discovered in time.
+While simple, this is not efficient, especially for Verifiers interested in a tiny subset of Claims.
+
+Map keys cannot be generated/discovered from a Map Checkpoint, thus Maps do not allow for efficient discoverability through key enumeration.
+However, if the expected location for Claims in a Map is precisely understood ahead of time by the Believer and Verifier, Maps can provide Claim discoverability.
+
+## Claim Subject Maps (CSMs)
+
+### New Terms
+
+<dl>
+<dt>Claim Subject</dt>
+<dd>the entity to which a Claim relates (also referred to as ‘Subject’, for brevity)</dd>
+
+<dt>Mog</dt>
+<dd>a Map where the values are Logs</dd>
+
+<dt>Claim Subject Map/Mog (CSM)</dt>
+<dd>this is a Map keyed by the Claim Subject. For Mogs, the value is a Log of Claims, for Maps the value is a single Claim. In either case, the Claims keyed under a Subject must relate to the Subject.</dd>
+</dl>
+
+### Introduction
+
+CSMs allow for more efficient discoverability of Claims, particularly in ecosystems where there are many independent Claim Verifiers with distinct Claim Subjects.
+Claim Verifiers can become somewhat ignorant and selfish, looking after only their own Subjects and trusting others to look after their own.
+The trade-off for this is the risk that any Claims placed in the map at locations the Verifier does not expect will not be discovered by the Verifier.
+This risk can be eliminated with careful design of Claim Subjects and how they are extracted from Claims.
+
+### Requirements
+
+To use a CSM for discoverability, these properties must hold:
+ 1. Claim Subjects can be derived from only the Claim. Examples (Claim Subjects in bold):
+    * CT: the Claim (cert) contains the **domain(s)** the certificate is for
```

Comment:
I'd be tempted to expand to Certificate Transparency.

Also, "golang" (or maybe "Golang")?